package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "workshop-api-testing-js",
+  "version": "1.0.0",
+  "description": "This is a Workshop about Api Testing in JavaScript",
+  "main": "index.js",
+  "scripts": {
```

Comment:
Add the engines key in the package.json

Reply:
Mi opinion sobre esto es similar, en esta primera parte se autogenero el package.json, esperaba que en otra sesión agregarle estos aspectos, me cuentas si estas de acuerdo?